http-server/src/main/java/com/facebook/airlift/http/server/HttpServer.java

```diff
@@ -358,7 +358,7 @@ public HttpServer(HttpServerInfo httpServerInfo,
         handlers.addHandler(createServletContext(theServlet, parameters, filters, tokenManager, loginService, "http", "https"));
 
         RequestLogHandler statsRecorder = new RequestLogHandler();
```

Comment:
How about:

```
if(config.isRequestStatsEnabled()){
  RequestLogHandler statsRecorder = new RequestLogHandler();
  statsRecorder.setRequestLog(new StatsRecordingHandler(stats));
  handlers.addHandler(statsRecorder);
}
```

?

Reply:
This is cleaner, I didn't know this handler is optional.  cc @wenleix